lib/data/US.yml

```diff
@@ -3680,7 +3672,7 @@
 '36865':
   :state_code: AL
   :state_name: Alabama
-  :city: Loachapoka
+  :city: LoachAPOka
```

Comment:
Is this a find and replace gone wrong? :/

Reply:
ugh yes 🤦‍♀

Reply:
fixed it